sssom/util.py

```diff
@@ -194,6 +194,8 @@ def parse(filename: str) -> pd.DataFrame:
 
 def collapse(df: pd.DataFrame) -> pd.DataFrame:
     """Collapse rows with same S/P/O and combines confidence."""
+    df[CONFIDENCE] = df[CONFIDENCE].replace("", np.nan)
+    df[CONFIDENCE] = df[CONFIDENCE].astype(float)
```

Comment:
Something like this should live where in to_mapping_set_data_frame, basically wherever a LinkML model gets converted to pandas (should be one place only)

Reply:
Done!